livestream/api_test.py

```diff
@@ -0,0 +1,14 @@
+"""livestream API tests"""
+from livestream.api import get_upcoming_events
+
+
+def test_get_upcoming_events(settings, mocker):
+    """test get upcoming events"""
+    settings.LIVESTREAM_ACCOUNT_ID = 392_239
+    settings.LIVESTREAM_SECRET_KEY = "secret key"
+    requests_patch = mocker.patch("requests.get", autospec=True)
+    get_upcoming_events()
+    requests_patch.assert_called_once_with(
```

Comment:
Can you also test the return value? `assert get_upcoming_events() == requests_patch.return_value` I think should work

Reply:
sure